### ISSUE TITLE
Once mozilla-central upgrades to json schema draft-04 remove the upgrade process

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-linter#1276]
Once [bug 1243516](https://bugzilla.mozilla.org/show_bug.cgi?id=1243516) is fixed the schema upgrade process that happens as part of the import can be removed.

**Comment by EnTeQuAk** [comment]
@muffinresearch ping if there's any way bringing this back to the teams attention maybe? The most recent schema updates went through nicely and weren't much work but that most certainly won't be the case all the time.

**Comment by muffinresearch** [comment]
> @muffinresearch ping if there's any way bringing this back to the teams attention maybe? The most recent schema updates went through nicely and weren't much work but that most certainly won't be the case all the time.

Yep, I've asked for a timeline here: https://bugzilla.mozilla.org/show_bug.cgi?id=1243516#c10

**Comment by willdurand** [comment]
Probably not something to worry about anymore.
